js/src/assessors/taxonomyAssessor.js

```diff
@@ -5,7 +5,7 @@ var keyphraseLength = require( "yoastseo/js/assessments/seo/keyphraseLengthAsses
 var keywordDensity = require( "yoastseo/js/assessments/seo/keywordDensityAssessment.js" );
 var metaDescriptionKeyword = require( "yoastseo/js/assessments/seo/metaDescriptionKeywordAssessment.js" );
 var MetaDescriptionLength = require( "yoastseo/js/assessments/seo/metaDescriptionLengthAssessment.js" );
-var titleKeyword = require( "yoastseo/js/assessments/seo/titleKeywordAssessment.js" );
+var TitleKeyword = require( "yoastseo/js/assessments/seo/titleKeywordAssessment.js" );
```

Comment:
The original variable name was written in camel case, but you changed it to pascal case. For consistency it would be better to keep everything camel case.

Reply:
I understand that it is a JS requirement to keep class names capitalised. This explains the accidental PascalCase.